engine/src/cubos/engine/gl/deferred/renderer.cpp

```diff
@@ -605,7 +605,7 @@ void deferred::Renderer::onRender(const Camera& camera, const Frame& frame, Fram
     {
         if (lightData.numSpotLights >= CUBOS_DEFERRED_RENDERER_MAX_SPOT_LIGHT_COUNT)
         {
-            logWarning("deferred::Renderer::onRender() failed: number of spot lights to be drawn "
+            CUBOS_WARN("Number of spot lights to be drawn "
                        "this frame exceeds CUBOS_DEFERRED_RENDERER_MAX_SPOT_LIGHT_COUNT ({}).",
```

Comment:
I'd rename this to not include the raw variable name:
```suggestion
                       "this frame exceeds the maximum allowed ({}).",
```